jenkins-x-bdd.yml

```diff
@@ -152,6 +152,10 @@ pipelineConfig:
                   - --suite-name
                   - "Static_BDD_Tests"
 
+              - name: clear-kubeconfig
```

Comment:
what's that good for?

Reply:
It's making sure we're no longer using the test cluster context when we subsequently run `jx step stash`.